app.json

```diff
@@ -4,6 +4,146 @@
   "keywords": ["events", "community"],
   "repository": "https://github.com/aaronpk/Meetable",
   "addons": [
-      "heroku-redis"
+    "heroku-redis",
+    {
+      "plan": "cleardb:ignite",
+      "as": "DATABASE"
+    }
+  ],
+  "env": {
```

Comment:
I understand this can be intimidating for a user. I'm happy to remove, but we need to know that it will harm the utility of the deploy button for some. (Me for example)

Reply:
I'm confused why this would make the deploy button less useful. I haven't actually tried this myself yet, but my thought was that by having zero options in the deploy button, hitting "deploy" would immediately drop you into the app and then the installer kicks in and walks you through everything. Assuming your database add-on is already there, then you'd never even see the database setup questions either, which is great.

Reply:
Right.

## Barriers

Barriers move from, can I type into a text input, to can I install the heroku cli; copy-paste some instructions without cutting off or missing letters. Not all Heroku users are comfortable with the heroku CLI tools.

## Utility

I can no-longer put everything in there to avoid the installer. I'm dragged into a screen that isn't necessary to my goal of having light-weight meeting software.

Ultimately it's not perfect either way. It's not a turn-key install; in that I cannot next->next->next with sane defaults either way.

I'll be chopping out the values for now as it's not the most pressing thing.